wiki/Main_Page/es.md

```diff
@@ -42,14 +39,14 @@ Todo sobre círculos, tambores, frutas y el sintetizador: elementos clave, conce
 
 [Modos de juego](/wiki/Game_mode): [osu!](/wiki/Game_mode/osu!) • [osu!taiko](/wiki/Game_mode/osu!taiko) • [osu!catch](/wiki/Game_mode/osu!catch) • [osu!mania](/wiki/Game_mode/osu!mania)
 
-[Beatmap](/wiki/Beatmap) • [Dificultad](/wiki/Beatmap/Difficulty) • [Objetos de golpeo](/wiki/Hit_object) • [Mods](/wiki/Game_modifier) • [Puntuación](/wiki/Gameplay/Score) • [Repetición](/wiki/Gameplay/Replay) • [Medallas](/wiki/Medals) • [Multi](/wiki/Client/Interface/Multiplayer)
+[Beatmap](/wiki/Beatmap) • [Dificultad](/wiki/Beatmap/Difficulty) • [Objetos](/wiki/Hit_object) • [Mods](/wiki/Game_modifier) • [Puntuación](/wiki/Gameplay/Score) • [Repetición](/wiki/Gameplay/Replay) • [Medallas](/wiki/Medals) • [Multi](/wiki/Client/Interface/Multiplayer)
 
 </div>
 <div class="wiki-main-page-panel">
 
 # [Editor de beatmaps](/wiki/Client/Beatmap_editor)
 
-Todos los beatmaps de osu! son creados por la imaginación desbordante de su comunidad dinámica. ¿Alguna vez quisiste compartir tu música favorita con otros, pero nunca supiste cómo? Familiarícese con el editor incorporado y las posibilidades que le ofrece.
+Todos los beatmaps de osu! son creados por la imaginación desbordante de su comunidad dinámica. ¿Siempre quisiste compartir tu música favorita con los demás, pero nunca supiste cómo? Familiarícese con el editor integrado y las oportunidades que ofrece.
 
 Secciones: [Composición](/wiki/Client/Beatmap_editor/Compose) • [Diseño](/wiki/Client/Beatmap_editor/Design) • [Tiempo](/wiki/Client/Beatmap_editor/Timing) • [Configuración de la canción](/wiki/Client/Beatmap_editor/Song_Setup)
 
```

Comment:
I would change `Divisor de golpes` and `Distancia de golpes` for `Division de ritmo` and `Distancia de ritmo` in the `Componentes` section here. I believe is the proper terminology we use [nowadays too](https://github.com/ppy/osu-wiki/blob/43c63c9cecca99d29e5e1d7dfe66daabceb59f2a/wiki/Client/Beatmap_editor/Beat_Snap_Divisor/es.md?plain=1#L5)

Reply:
Should be *Divisor, my bad.